[MM-54]: Added the test cases for revoking the application on gitlab and proper message is displayed for member access request in todo. (#27)
Brightscout/mattermost-test-management: data/test-cases/plugins/gitlab/sidebar/merge_request.md

Reviewer: According to the PR description this file should not be here

Author: updated.

---
# (Required) Ensure all values are filled up
name: "Merge request message is rendered properly in the RHS of gitlab on MM"
status: Active
priority: Normal
folder: Sidebar
authors: "@arush-vashishtha"
team_ownership: []
priority_p1_to_p4: P2 - Core Functions (Do core functions work?)

# (Optional)
location: null
component: null
tags: []
labels: []
tested_by_contributor: null

# (Optional) Test type and tools
cypress: null
detox: null
mmctl: null
playwright: null
rainforest: []
manual_test_environments: []

# Do not change
id: null
key: null
created_on: null
last_updated: null
case_hashed: null
steps_hashed: null
---

**Step 1**

> 1. Generate a access request to your desired private project/group from another user on Gitlab.

Reviewer: ```suggestion
1. Generate an access request to your desired project/group from another user on Gitlab.
```

2. Navigate to MM and click on the `To-do list` icon in the LHS menu on MM.
3. Check the desired member access request in the RHS of `To-do list` on MM.

**Step 2**

1. Accept or reject any desired member access request on Gitlab.
2. Navigate to MM and click on the `To-do list` icon in the LHS menu on MM.
3. Check the desired member access request in the RHS of `To-do list` on MM.

**Expected**

> The member access request to a project/group should show the description of the request in proper UI in RHS of Gitlab on MM.

Reviewer: ```suggestion
The member access request to the project/group should show the description of the request in proper UI in RHS of Gitlab on MM.
```

> The member access request to a project/group should show the description of the request in proper UI on MM.

Reviewer 2: in RHS
